cmd/manager/main.go

```diff
@@ -118,6 +118,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Define a kubeClient for any processes that need to run during operator startup or independent routines to use
+	kubeClient, err := client.New(cfg, client.Options{})
+	if err != nil {
+		log.Error(err, "Failed to create a kubernetes client")
+		os.Exit(1)
+	}
+
+	errors := utils.InitControllerMaxReconciles(kubeClient)
```

Comment:
We needed to move it up before the controllers init.  We could pull out the get configmap stuff and move that into the main function, but I think it will be easier to follow just leaving it as passing in the kubeClient.  The `initOperatorConfigMapVars(kubeClient)` function just takes the kubeClient as well, so at least it’s consistent.

Reply:
when testing in crc we were seeing an error about being unable to get the shard name from the configmap when initializing AAO. Moving the configmap around fixed that